content/zh-cn/case-studies/ygrene/index.html

```diff
@@ -1,130 +1,187 @@
 ---
 title: 案例研究：Ygrene
-
+linkTitle: Ygrene
 case_study_styles: true
 cid: caseStudies
-css: /css/style_ygrene.css
+logo: ygrene_featured_logo.png
+featured: true
+weight: 48
+quote: >
+  我们必须改变一些实践和代码，以及构建的方式，但我们能够在一个月左右的时间内将我们的主要系统安装到 Kubernetes 上，然后在两个月内投入生产。这对于金融公司来说是非常快的。
+
+new_case_study_styles: true
+heading_background: /images/case-studies/ygrene/banner1.jpg
+heading_title_logo: /images/ygrene_logo.png
+subheading: >
+  Ygrene: 使用原生云为金融行业带来安全性和可扩展性
+case_study_details:
+  - 公司: Ygrene
+  - 地点: 佩塔卢马，加利福尼亚州
```

Comment:
```suggestion
  - 地点: Petaluma, Calif.
```
Location does not require translation.